src/process_results.py

```diff
@@ -157,6 +157,8 @@ def add_missing_entries(record: dict) -> dict:
         record["few_shot"] = True
     if "generative" not in record:
         record["generative"] = False
+    if "scandeval_version" not in record:
+        record["scandeval_version"] = "<9.2.0"
```

Comment:
This has been renamed to "euroeval_version" now, so we should check for both of them:

```suggestion
    if "euroeval_version" not in record and "scandeval_version" not in record:
        record["euroeval_version"] = "<9.2.0"
```